OpenRA.Mods.RA/Traits/GpsWatcher.cs

```diff
@@ -77,12 +83,18 @@ void Refresh()
 		void RefreshInner()
 		{
 			var wasActive = Active;
+			var allyWatchers = owner.World.ActorsWithTrait<GpsWatcher>().Where(kv => kv.Actor.Owner.IsAlliedWith(owner));
```

Comment:
now that we've ditched fragile alliances, yes, but again I wanted to minimize the changes for the sake of getting this this immediately before the playtest.  This is the least of the remaining problems in the GPS code.